### DISCUSSION TITLE
next.config.js file gets altered when deploying

### DISCUSSION THREAD

**Discussion by lauroaraujo** [vercel/vercel#4260] [General]
After changing my build command to a custom npm script `yarn now:build`, which just calls standardjs to lint the code before deploying (`standard && next build`), my build started breaking with linter errors on the `next.config.js` file.

Nothing breaks locally, and the errors reported do not match the `next.config.js` file in our repositories.

This happens both in the github integration and deploying locally with `now`

**Does Now changes the `next.config.js` file in any way before deploying? Is this documented somewhere?**

**Comment by coetry** [comment]
Hi @lauroaraujo we'll gladly take a look at your issue. Can you please send us an email to support@vercel.com with deployment URLs so we can take a look at your build logs? 

We don't modify `next.config.js` files when deploying.
